pkg/chaosdaemon/httpchaos_server.go

```diff
@@ -100,19 +108,19 @@ func (s *DaemonServer) ApplyHttpChaos(ctx context.Context, in *pb.ApplyHttpChaos
 
 	req, err := http.NewRequest(http.MethodPut, "/", bytes.NewReader(config))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("fail to create request: %v", err)
```

Comment:
Please use `"github.com/pkg/errors"` `erorrs.Wrapf()` instead of `fmt.Errorf`

Reply:
Fixed